### DISCUSSION TITLE
[Idea] <In search of a dev to partner with for Optimism Art Contest>

### DISCUSSION THREAD

**Discussion by DeanKrupka** [ethereum-optimism/developers#122] [Ideas]
### Idea

I'm hoping to create something cool to enter Optimism's Art Contest in January -- but to do it right, I'd need some more technical know-how than I currently have (I'm only 2-months into learning Solidity). If there's a builder out there who'd be down to partner for an onchain art experiment using EAS on Optimism, please lmk! You can read a sketch of my plan here: 

https://docs.google.com/document/d/1jXSe_vT53ftCQlKjV6reNVQDBtylUKU7ZVTwe7Q6L8g/edit

**Comment by sbvegan** [comment]
Interesting concept! I'd recommend posting in our discord too. You might be able to find someone interested there :)

**Reply by DeanKrupka** [reply]
I posted in the #builder-general channel of the OP discord.  Hoping someone has some available time to assist.  Thanks for giving it a look!
